lib/dependents.rb

```diff
@@ -0,0 +1,5 @@
+require_relative 'game.rb'
+require_relative 'team.rb'
+require_relative 'game_teams.rb'
+require_relative 'hashable.rb'
+require_relative 'calculable'
```

Comment:
Let's have a conversation for a future refactor about whether we should include extensions on file names in require statements. Don't think I have strong feelings either way, just think we should be consistent across classes.